src/Core/Resources/Infrastructure/Repository/DbReadResourceRepository.php

```diff
@@ -171,7 +171,8 @@ private function generateFindResourcesRequest(
         FROM `:dbstg`.`resources`
         LEFT JOIN `:dbstg`.`resources` parent_resource
             ON parent_resource.id = resources.parent_id
-        LEFT JOIN `:dbstg`.`severities`
+            AND parent_resource.type = ' . self::RESOURCE_TYPE_HOST .
```

Comment:
Are all parents of the Host type?

Reply:
For now yes and BA will not have their BV parentality when it will be integrated